docs/toml.md

```diff
@@ -155,11 +155,16 @@ Supported filters:
 ## Access log definition
 
 The standard access log uses the textual Common Log Format (CLF), extended with additional fields.
-Alternatively logs can be written in JSON. 
-Using the default CLF option is simple, e.g.
+Alternatively logs can be written in JSON.
+By default logs will be written to stdout and use the CLF format.
```

Comment:
Can you change this part in order to understand that without the `[accessLog]` section, we will not have any accesslog ?

Moreover can you add the same information in the `traefik.sample.toml`.